core/src/dird/director_jcr_impl.h

```diff
@@ -78,6 +79,7 @@ struct Resources {
   directordaemon::PoolResource* next_pool{};    /**< Next Pool used for migration/copy and virtual backup */
   directordaemon::FilesetResource* fileset{};   /**< FileSet resource */
   directordaemon::CatalogResource* catalog{};   /**< Catalog resource */
+  directordaemon::runtime_job_status_t* rjs{};  /**< Runtime Job Status. May point to the rjs of another resource (e.g. for consolidation vf jobs this points to the rjs of the parent consolidation job's resource) */
```

Comment:
I used to have it in the DirectorJcrImpl at first, but then moved it to the Resources struct because I thought it would be a better fit as the rjs is owned by a JobResource.
Should I move it back to DirectorJcrImpl?